pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -1666,7 +1666,7 @@ default-package-overrides:
   - prettyprinter-compat-wl-pprint ==1.0.0.1
   - prettyprinter-convert-ansi-wl-pprint ==1.1
   - pretty-show ==1.9.5
-  - pretty-simple ==2.2.0.1
+  - pretty-simple ==3.2.1.0
```

Comment:
Checkout the comment on the top of this list.

Also, I think `pretty-simple-2.2.0.1` should be fine here, since it still contains the executable.

Reply:
yes, it currently still builds 2.2.0., I didn't regenerate Haskell packages

but I want it to be the latest version, so I just updated this file so maybe someone in future will regenerate them for me

what do you think?

Reply:
I have split commits

Reply:
The Haskell packages in nixpkgs generally track the latest stackage LTS release.  Unless there is some really good reason, we generally don't bump versions past what is given to us in the Stackage LTS.

I don't think there is any big reason to bump `pretty-simple` past `2.2.0.1` right now.

If you remove this change here in ` configuration-hackage2nix.yaml`  I'll merge this in.

Reply:
done